firewood/src/merkle.rs

```diff
@@ -31,6 +31,8 @@ pub type Value = Box<[u8]>;
 
 // convert a set of nibbles into a printable string
 // panics if there is a non-nibble byte in the set
+// TODO: This is more efficient than the LowerHex implementation for Path
```

Comment:
"more efficient" is debatable. This allocates a string whereas the lowerhex writes to whatever the formatter is wrapping, which could be a string buffer or could be directly to a writer. If this is constructed only to immediately write to another buffer, then this is not more efficient.